### PR TITLE
Fixes the issues(missing variable) from regression tests, adds minor updates from the template.

### PR DESCRIPTION
**Pull request by krishr2d2**
Opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by krishr2d2** [comment]
There's a `TODO` item in the `Model registration custom component` subsection of this notebook. The final model is supposed to be registered to the `Model Registry` in it.

File: notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb [hunk 1]
```diff
@@ -0,0 +1,2444 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Change from master branch to main branch</p><p> </p><p>Add H1 title above the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Split into 4 text cells</p><p> </p><p>Move Dataset after Objective</p><p> </p><p>Use template for Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p><p> </p><p>Combine the 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>use notebook template</p><p> </p><p>combine all code installs into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>use notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p><p> </p><p>Combine the 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>use latest template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>use template version for REGION and BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Combine these into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>no 1st person, use 2nd person</p><p> </p><p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>combine the 2 code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>combine this and next code cell.</p><p> </p><p>Add text cell explaining</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Initiate -&gt; Initialize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>no future tense, remove 'will'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Lots of code! Need to add heading and some explanation of what the code does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Add heading and explain code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Add heading and explain code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>Explain the component and params.</p><p> </p><p>Params should be on single line. Ex.</p><p> </p><p>def func(param1: str,</p><p>               paramN: str</p><p>):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>explain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>explain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>explain steps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb:1]: <p>ok, the pipeline finished. Example is hanging. Need some code/text cells that check that the pipeline did something, like create the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**